Assets/Scripts/Models/EventAction.cs

```diff
@@ -123,15 +123,25 @@ public void Deregister(string actionName, string luaFunc)
     /// <param name="actionName">Name of the action being triggered.</param>
     /// <param name="target">Object, passed to LUA function as 1-argument (TODO: make it an object).</param>
```

Comment:
Also, that's a terrible place to have put a TODO, I didn't even see it... it took me a bit to even spot it after you mentioned it directly.